datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -473,15 +473,15 @@ async fn roundtrip_inlist_5() -> Result<()> {
     // on roundtrip there is an additional projection during TableScan which includes all column of the table,
     // using assert_expected_plan here as a workaround
     assert_expected_plan(
-    "SELECT a, f FROM data WHERE (f IN ('a', 'b', 'c') OR a in (SELECT data2.a FROM data2 WHERE f IN ('b', 'c', 'd')))",
-    "Projection: data.a, data.f\
-     \n  Filter: data.f = Utf8(\"a\") OR data.f = Utf8(\"b\") OR data.f = Utf8(\"c\") OR Boolean(true) IS NOT NULL\
-     \n    Projection: data.a, data.f, Boolean(true)\
-     \n      Left Join: data.a = data2.a\
-     \n        TableScan: data projection=[a, f]\
-     \n        Projection: data2.a, Boolean(true)\
-     \n          Filter: data2.f = Utf8(\"b\") OR data2.f = Utf8(\"c\") OR data2.f = Utf8(\"d\")\
-     \n            TableScan: data2 projection=[a, f], partial_filters=[data2.f = Utf8(\"b\") OR data2.f = Utf8(\"c\") OR data2.f = Utf8(\"d\")]",
+        "SELECT a, f FROM data WHERE (f IN ('a', 'b', 'c') OR a in (SELECT data2.a FROM data2 WHERE f IN ('b', 'c', 'd')))",
```

Comment:
I think ut makes sense to show a physical plan for mark join as well

Reply:
Added in https://github.com/apache/datafusion/pull/13134/commits/3c72e374e92e31fbea6b91aa82dfd6aa585da67d